foundations/src/telemetry/server.rs

```diff
@@ -2,7 +2,7 @@
 use super::metrics;
 use super::settings::TelemetrySettings;
 use crate::{BootstrapResult, Result};
-use anyhow::anyhow;
+use anyhow::{anyhow, Context as AnyhowContext};
```

Comment:
Nit: you don't really need an alias for traits that you just want to bring in scope: `Context as _`.

Reply:
Ah, that is good to know.